aubreylib/resource.py

```diff
@@ -215,7 +215,7 @@ def get_metadata_file(self, parsed_mets):
         else:
             raise ResourceObjectException("Descriptive metadata not defined " +
                                           "in the METS file.")
-        for attribs, value in mdRef.attrib.items():
+        for attribs, value in list(mdRef.attrib.items()):
```

Comment:
What does converting this to a `list` do?

Reply:
It's a thing the futurize tool does automatically. I missed removing it here.